src/main/java/org/openzal/zal/extension/Utils.java

```diff
@@ -0,0 +1,5 @@
+package org.openzal.zal.extension;
+
+public class Utils {
```

Comment:
This class is empty due to Draft PR?

Reply:
I have to check this, don't know why it is here :thinking:  Thanks!

Reply:
Removed